ChangeLog

```diff
@@ -1,6 +1,16 @@
 Change Log
 ==========
 
+Version 2.1.3.0.0.1 - 2017-07-07 - Jonathan W. Zaleski
```

Comment:
I think the versioning info can be removed, we'll update to the next minor version in the next release.

Reply:
Just to confirm, you're recommending removing it throughout? Would you like me to remove the "Bug fixes" snippet as well? Happy to do so if it helps get this merged/released.